public/index.tpl

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Wobscale.chat password reset</title>
+    <script>
+      
+    </script>
+  </head>
+  <body>
+    <h1>So, you forgot your password</h1>
+
+    <h2> {{ notice }} </h2>
```

Comment:
It's possible if I do

```
{{#somestruct}}
  <h2> {{ .notice }}
{{/#somestruct}}
```

Just didn't bother with defining a struct like that.

It doesn't show as anything if it's empty so :shrug:.